chapters/statements.tex

```diff
@@ -299,6 +303,8 @@ \subsection{While-Statement}\label{while-statement}
 \item
   If the \lstinline!expression! of the \lstinline!while!-statement is true, the entire body of the \lstinline!while!-statement is executed (except if a \lstinline!break!-statement, see \cref{break-statement}, or a \lstinline!return!-statement, see \cref{return-statements}, is executed), and then execution proceeds at step~\ref{while-execution-top}.
 \end{enumerate}
+Neither the \lstinline!expression! nor any expression in the body shall generate events.
```

Comment:
Paragraph break:
```suggestion

Neither the \lstinline!expression! nor any expression in the body shall generate events.
```

In addition to this one, it would be nice to also get a paragraph break before this line:
```
The \lstinline!while!-statement corresponds…
```
(It's funny that we compare with _programing languages_, as if Modelica wasn't a programming language, and as if every programming language has a while loop construct.)

Reply:
Also use standard formulation:
```suggestion

Event-generating expressions are neither allowed in the \lstinline!expression! nor in the loop body statements.
```

Reply:
> In addition to this one, it would be nice to also get a paragraph break before this line:
> 
> ```
> The \lstinline!while!-statement corresponds…
> ```
> 
> (It's funny that we compare with _programing languages_, as if Modelica wasn't a programming language, and as if every programming language has a while loop construct.)

I added line break and changed to _other_ programming languages.